junwoo/controllers/userController.js

```diff
@@ -0,0 +1,23 @@
+//controllers/userController.js
+
+const userService = require("../services/userService");
+
+const signUp = async (req, res) => {
+  try {
+    const { name, email, password, profileImage } = req.body;
+    if (!name || !email || !password || !profileImage) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await userService.signUp(name, email, password, profileImage);
+
+    res.status(201).json({ message: "SIGNUP_SUCCESS" });
+  } catch (err) {
+    console.log(err);
```

Comment:
에러가 발생했을 때 해당 에러에 대한 로그를 확인하기 위한 좋은 코드네요!👍

다만 다른 로그와 겹쳐서 특정 로그를 파악하기 어려울 수도 있으니, 아래와 같이 작성해보시고 그 차이를 확인해주세요!
```suggestion
    console.error(err);
```